src/reduxForm.js

```diff
@@ -255,6 +255,7 @@ const createReduxForm =
 
           render() {
             // remove some redux-form config-only props
+            /* eslint-disable no-unused-vars */
```

Comment:
Here we use destructuring to compute `passableProps`. We need this to mute ESLint because we create a lot of unused bindings.